airflow-core/src/airflow/api_fastapi/execution_api/routes/dag_runs.py

```diff
@@ -150,3 +155,31 @@ def get_dagrun_state(
         )
 
     return DagRunStateResponse(state=dag_run.state)
+
+
+@router.get(
+    "/{dag_id}/count-by-run-ids-and-states",
```

Comment:
```suggestion
    "/{dag_id}/count
```

Let's keep it simple so we can add more query params in future

Reply:
Make sense, updated now :)